base/int.jl

```diff
@@ -204,9 +201,9 @@ rem{T<:Integer}(x::T, ::Type{T}) = x
 rem(x::Integer, ::Type{Bool}) = ((x&1)!=0)
 mod{T<:Integer}(x::Integer, ::Type{T}) = rem(x, T)
 
-convert{T<:UnionS64Types,Tf<:Union{Float32,Float64}}(::Type{T}, x::Tf) =
+convert{T<:BitSigned64,Tf<:Union{Float32,Float64}}(::Type{T}, x::Tf) =
     box(T,checked_fptosi(T,unbox(Tf,x)))
-convert{T<:UnionU64Types,Tf<:Union{Float32,Float64}}(::Type{T}, x::Tf) =
+convert{T<:BitUnsigned64,Tf<:Union{Float32,Float64}}(::Type{T}, x::Tf) =
     box(T,checked_fptoui(T,unbox(Tf,x)))
```

Comment:
@JeffBezanson fyi: the rename on these lines will cause a rebase conflict with the correction to the function signature at: https://github.com/JuliaLang/julia/pull/13412/files#diff-d5b98c05f9bca9e586d65cfc16c83833R207